Source/Diagnostics/ComputeDiagFunctors/JdispFunctor.cpp

```diff
@@ -59,8 +56,11 @@ JdispFunctor::operator() (amrex::MultiFab& mf_dst, int dcomp, const int /*i_buff
         -1, *m_mf_j, 0, 0, 1, Jdisp.nGrowVect()
     );
 
-   // Subtract the interpolated j_external value from j_displacement.
     if (hybrid_pic_model) {
+        // Subtract the interpolated j_external value from j_displacement.
+        /** pointer to external currents (Jext) multifab */
+        amrex::MultiFab* m_mf_j_external = hybrid_pic_model->get_pointer_current_fp_external(m_lev, m_dir);
```

Comment:
Thanks for this PR, @roelof-groenewald ! Maybe we could change the name of this variable to `mf_j_external`, since it is not a member variable.

Reply:
Good point. I'll change that now. Thanks!